components/raftstore/src/store/peer_storage.rs

```diff
@@ -2118,7 +2118,7 @@ mod tests {
     ) -> PeerStorage<KvTestEngine, RaftTestEngine> {
         let mut store = new_storage(region_scheduler, raftlog_fetch_scheduler, path);
         let mut write_task = WriteTask::new(store.get_region_id(), store.peer_id, 1);
-        store.append(ents[1..].to_vec(), &mut write_task);
+        store.append(ents.to_vec(), &mut write_task);
```

Comment:
Better change the case instead of changing it here as it makes no sense to test impossible operations.

Reply:
What do you mean? I didn’t know the entries stored in raftdb are inconsistent with raft state by nature.

Reply:
What I mean is remove all entries prior to (3, 3) in test cases. entries should be consistent with raft state and no entries prior to truncated state can be appended.